srcpkgs/freetype/template

```diff
@@ -12,6 +12,15 @@ license="GPL-2.0-or-later, FTL"
 homepage="https://www.freetype.org/"
 distfiles="${NONGNU_SITE}/${pkgname}/${pkgname}-${version}.tar.xz"
 checksum=1543d61025d2e6312e0a1c563652555f17378a204a61e99928c9fcef030a2d8b
+build_options="cleartype"
+desc_option_cleartype="Enable Microsoft's subpixel rendering (ClearType)"
+build_options_default="cleartype"
```

Comment:
Freetype itself says [emphasis added]
> Note that this feature is covered by several Microsoft patents and **should not be activated in any default build of the library**.

A build option is okay, but this should not be on by default.

Reply:
See #13296, it became available due to MS joining OIN.

Reply:
Thanks for the clarification. I didn't look at the contents of the removed patch to see that we're already enabling this option. In that case, no harm in making it a default option.